azure-pipelines.yml

```diff
@@ -37,8 +37,22 @@ jobs:
         - AndroidEasingFunctions
         - Square
         - JakeWharton.Picasso2OkHttp3Downloader
+      preBuildSteps:
+        - pwsh: |
+            dotnet tool uninstall --global Cake.Tool
+            dotnet tool install --global Cake.Tool
```

Comment:
I also added a few parameters to the ci script:
https://github.com/xamarin/XamarinComponents/blob/master/.ci/build.yml#L17-L26

so for cake you can also override the `cake` parameter with the version you want instead of installing it.

Reply:
uninstall/install is my pattern for update/upgrade and I have had positive experience with it.

Reply:
If you are more comfortable wit this way, then no biggie.